src/ListTabulator/index.ts

```diff
@@ -802,26 +976,55 @@ export default class ListTabulator {
    * @param {boolean} atStart - where to set focus: at the start or at the end
    * @returns {void}
    */
-  focusItem(item: Element, atStart: boolean = true): void {
+  focusItem(item: ListItemElement, atStart: boolean = true): void {
     const itemContent = item.querySelector<HTMLElement>(
       `.${DefaultListCssClasses.itemContent}`
     );
     if (!itemContent) {
       return;
     }
 
-    Caret.focus(itemContent, atStart);
+    focus(itemContent, atStart);
   }
 
   /**
    * Get out from List Tool by Enter on the empty last item
-   *
+   * @param index - optional parameter represents index, where would be inseted default block
    * @returns {void}
    */
-    getOutOfList(): void {
-      this.currentItem?.remove();
+  getOutOfList(index?: number): void {
```

Comment:
index seems redundant here, getOutOfList should always work same way: focus insert and block below

Reply:
still actual

Reply:
yep, it makes inset after `currentBlock` the problem is that in some cases i need to insert new block before current, and i use index for that purposes